chain/chain/src/test_utils.rs

```diff
@@ -55,12 +54,10 @@ use near_chain_configs::ProtocolConfig;
 use near_primitives::block_header::{Approval, ApprovalInner};
 use near_primitives::state_record::StateRecord;
 
-#[derive(
-    BorshSerialize, BorshDeserialize, Serialize, Hash, PartialEq, Eq, Ord, PartialOrd, Clone, Debug,
-)]
+#[derive(BorshSerialize, BorshDeserialize, Hash, PartialEq, Eq, Ord, PartialOrd, Clone, Debug)]
 struct AccountNonce(AccountId, Nonce);
 
-#[derive(BorshSerialize, BorshDeserialize, Serialize, Clone, Debug)]
+#[derive(BorshSerialize, BorshDeserialize, Clone, Debug)]
```

Comment:
why we even derived `Serialize` for something that is only used in tests is beyond me :(